Update cost_details.md (#31285)
DataDog/documentation: content/en/account_management/plan_and_usage/cost_details.md

- Allocate costs according to their source
- Gain insight into how costs are tracking

> To gain visibility into daily Datadog spending in Cloud Cost Explorer, dashboards and [cost monitors][7], see [Datadog Costs in Cloud Cost Management][8].

Reviewer: ```suggestion
For visibility into daily Datadog spending in Cloud Cost Explorer, dashboards, and [cost monitors][7], see [Datadog Costs][8] under Cloud Cost Management.
```
